scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo starting airflow with command:
+echo airflow $@
+
+exec airflow $@
```

Comment:
It would be nice if this docker image would work with other commands, such as `bash` or `python` https://github.com/puckel/docker-airflow/blob/master/script/entrypoint.sh#L70-L95

Reply:
@ashb You can always do something like this: ` docker run --entrypoint bash -ti <image>`